tests/integration/docker-compose.yml

```diff
@@ -7,11 +7,30 @@ postgresdb:
   image: postgres:latest
   ports:
     - 5432
+  environment:
+    POSTGRES_PASSWORD: 'hunter2'
```

Comment:
The Docker image for Postgres at some point started requiring this.  I don't think I've run integration tests via docker-compose since then.